easybuild/easyconfigs/m/msgpack-c/msgpack-c-3.3.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,30 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/04
+easyblock = 'CMakeMake'
+
+name = 'msgpack-c'
+version = '3.3.0'
+
+homepage = 'http://msgpack.org/'
+description = """MessagePack is an efficient binary serialization format, which lets you exchange
+data among multiple languages like JSON, except that it's faster and smaller.   
+Small integers are encoded into a single byte while typical short strings       
+require only one extra byte in addition to the strings themselves."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [GITHUB_SOURCE]
+github_account = 'msgpack'
+sources = ['cpp-%(version)s.tar.gz']
```

Comment:
I wonder if this 
<https://github.com/msgpack/msgpack-c/releases/download/cpp-3.3.0/msgpack-3.3.0.tar.gz>
would be a better choice for the sources. It seems like the one they actually "promote" while the other is just a automatic git tag thing